okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/LeakDetector.java

```diff
@@ -84,15 +84,15 @@ private synchronized void pruneAndLog() {
     private void logLeak(Optional<RuntimeException> stackTrace) {
         if (stackTrace.isPresent()) {
             log.warn("Resource leak detected - did you forget to close a resource properly? "
-                            + "This will likely hurt performance. Stack trace is of the call where "
-                            + "the acquire happened.",
```

Comment:
@dansanduleac extra request-  could we try and keep this old string wrapping indentation?? I think it makes it easier to differentiate this first arg (even though it spans multiple lines) from the second and third args.

Reply:
If we could avoid modifying wrapped strings that aren't super incorrect, it would help some older codebases with longer wrapped templates. Take SQL for example:

```java
// If this gets re-wrapped, it becomes more difficult to read
PreparedStatement preparedStatement = connection.prepareStatement(""
    + "SELECT Orders.OrderID, Customers.CustomerName, Shippers.ShipperName "
    + "FROM Orders "
    + "INNER JOIN Customers ON Orders.CustomerID = Customers.CustomerID "
    + "INNER JOIN Shippers ON Orders.ShipperID = Shippers.ShipperID "
    + "WHERE Orders.OrderID = ?");
```

Reply:
So far I think the eclipse formatter won't reflow _any_ strings.